printing/printer.ml

```diff
@@ -745,28 +756,28 @@ let pr_subgoals ?(pr_first=true) ?(diffs=false) ?os_map
       begin
 	let exl = Evd.undefined_map sigma in
 	if Evar.Map.is_empty exl then
-	  (str"No more subgoals." ++ print_dependent_evars None sigma seeds)
+          (v 0 (str"No more subgoals." ++ pr_evar_info None sigma seeds))
```

Comment:
Indentation seems incorrect here.

Reply:
This has a tab to space conversion.  How many spaces to a tab character?

Reply:
Looks like github thinks it's 6, which is a strange choice.